### PR TITLE
added possibility to configure commands by passing an array

### PR DESCRIPTION
**Pull request by zolex**
It's now possible to configure a migration command by passing a configuration array.

### CODE REVIEW

**Comment by mikeSimonson** [comment]
@zolex Can you rebase your PR and add a test ?

Thanks

**Comment by kdubuc** [comment]
any news on this ?

File: lib/Doctrine/DBAL/Migrations/MigrationException.php [hunk 1]
```diff
@@ -63,4 +63,14 @@ public static function configurationFileAlreadyLoaded()
     {
         return new self(sprintf('Migrations configuration file already loaded'), 8);
     }
+    
+    public static function directoriesConfigInvalid()
+    {
+        return new self('Migraion directories config must be an array', 9);
```
Comment (mikeSimonson) [inline lib/Doctrine/DBAL/Migrations/MigrationException.php]: Migration instead of Migraion

**Comment by mikeSimonson** [comment]
To be merged the part in the abstractCommand needs to be merged into the ConfigurationHelper and it needs tests.

**Comment by mikeSimonson** [comment]
Another PR was merged with the feature cleaned up from this PR.

Thanks

**Pull request closed without merging**